packages/react-instantsearch/src/components/Highlight.js

```diff
@@ -3,7 +3,13 @@ import React from 'react';
 import Highlighter from './Highlighter';
 
 export default function Highlight(props) {
-  return <Highlighter highlightProperty="_highlightResult" {...props} />;
+  return (
+    <Highlighter
+      highlightProperty="_highlightResult"
+      widgetClassName="Highlight"
```

Comment:
We can adopt the same strategy as on the other widgets. We could bind the `classNames` from the wdiget and then pass `cx` down the tree (`Widget` > `Hightlight` > `Hightlighter`). We can also do it for the `Snippet` widget.